src/triton_pipeline/config/object_recognition.yaml

```diff
@@ -0,0 +1,10 @@
+pipeline_manager:
```

Comment:
As per a previous comment, the launch file should make it so that we don't need this anymore